pluggy/hooks.py

```diff
@@ -257,20 +257,29 @@ def __call__(self, *args, **kwargs):
                 )
         return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
 
-    def call_historic(self, proc=None, kwargs=None):
-        """ call the hook with given ``kwargs`` for all registered plugins and
+    def call_historic(self, result_callback=None, kwargs=None, proc=None):
+        """Call the hook with given ``kwargs`` for all registered plugins and
         for all plugins which will be registered afterwards.
 
-        If ``proc`` is not None it will be called for for each non-None result
-        obtained from a hook implementation.
+        If ``result_callback`` is not ``None`` it will be called for for each
+        non-None result obtained from a hook implementation.
```

Comment:
Please document that `proc` is deprecated in the docstring as well.

Reply:
Will do!